functional/vfio/run.sh

```diff
@@ -92,6 +100,10 @@ check_eth_dev() {
 	sudo -E ctr t exec --exec-id 2 "${container_id}" ip a | grep "eth"
 }
 
+get_dmesg() {
+	sudo -E ctr t exec --exec-id 2 "${container_id}" dmesg
```

Comment:
Might be worth creating a `get_ctr_cmd_output()` function that calls:

```bash
sudo -E ctr t exec --exec-id 2 "${container_id}" $cmd
```

... to avoid the duplication here (see `get_vfio_path()` above).

Reply:
ack, thanks